src/server/import-controller.js

```diff
@@ -12,7 +12,7 @@ import { getImportsQuerySchema, postImportBatchBodySchema } from '../domain/sche
 
 const { NotFound } = httpErrors;
 
-const jsonParser = express.json();
+const jsonParserLargePayload = express.json({ limit: '2MB' });
```

Comment:
Cool, what was the size of the batch that errored out in your case?

Reply:
It was almost 200kb, I saw we have this 2 MB for the docs as well so I took that. We could lower it, but I wouldn't overfit.

Reply:
No no, actually wanted to make sure we don't soon reach this limit again, all good.